README.md

```diff
@@ -763,6 +763,7 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 
 | Status | Device(s) | Name | Release | Notes |
 | :----: | :-------: | ---- | :-----: | :---- |
+| :yellow_heart:  | STM32U073C8<br>STM32U073CB<br>STM32U073CC<br>STM32U083CC | Generic Board | **2.11.0** |  |
```

Comment:
Could you spot in 2 lines: STM32U073C8<br>STM32U073CB<br>STM32U073CC then STM32U083CC after the STM3273R*